.ci/build-kit/compile.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
Currently, the steps to build utils, which this workflow depends upon is being done in the base image generation. If for example `ev-cli` needs to be updated for changes in this PR to work, there should be a local option/step to also build and install `ev-cli` and/or other tools.

Maybe it would not hurt to add a line to do that here?